chromium_src/chrome/browser/ui/views/frame/system_menu_model_builder.h

```diff
@@ -0,0 +1,29 @@
+/* Copyright (c) 2024 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_CHROMIUM_SRC_CHROME_BROWSER_UI_VIEWS_FRAME_SYSTEM_MENU_MODEL_BUILDER_H_
+#define BRAVE_CHROMIUM_SRC_CHROME_BROWSER_UI_VIEWS_FRAME_SYSTEM_MENU_MODEL_BUILDER_H_
+
+#include "chrome/browser/ui/views/frame/system_menu_model_delegate.h"
+
+#define Init                                \
+  UnUsed();                                 \
+                                            \
+ private:                                   \
+  friend class BraveSystemMenuModelBuilder; \
```

Comment:
@goodov 
Is it okay to do this kind of stuff to add a class as a friend? or We can go with patch(with macro) or something?

Reply:
yes, it's fine. we do this sometimes in other parts.

Reply:
Cool!